ForgeRock Cloud APIs.postman_collection.json

```diff
@@ -319,7 +314,7 @@
 								"header": [
 									{
 										"key": "Authorization",
-										"value": "Basic OTE3OTQ5YjExYzQ0MTU5ZjBlNjlmYzY5ZjE3MDJlYWI6UHluNktoU1J3VUt3anRocWF5ZU5ZeTBheG5BSTFTdTNGRjI4Zm9zM3VJQT0=",
+										"value": "Basic BASE_64_ENCODED_STRING",
```

Comment:
Can we turn this into an enviro variable?
intermediate = CryptoJS.enc.Base64.parse(base64content); // CryptoJS is an inbuilt object, documented here: https://www.npmjs.com/package/crypto-js

Reply:
Next step, https://trello.com/c/0HFohPy2

My first step is to make the REST calls / docs understandable to a target audience of devs who do *not* understand identity.

Reply:
Nice!

Reply:
Beau, I was really trying to get the base64 string to work within postman and almost had something working.  Maybe we can revisit with your variable idea.

Reply:
FWIW, I figured this out: in Postman, for each relevant REST call

* Set up "Basic Auth" under Authorization
* Include client_id as Username, client_secret as Password

With these steps, we don't need BASE_64_ENCODED_STRING